source/kubernetes/get-started/tutorials/app-update-deploy-monitor-logs/index.html.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Deploy a new version of the example test app, check the deployment, monitor the app on Grafana and view application logs
+title: Update, Deploy and Monitor an Application
```

Comment:
Super nit-picky but we should use sentence case [as per style guide](https://www.gov.uk/guidance/style-guide/a-to-z):

> Capitalisation
> (…) Always use sentence case, even in page titles and service names. 

```suggestion
title: Update, deploy and monitor an application
```